mpi4jax/__init__.py

```diff
@@ -1,4 +1,4 @@
-import mpi4jax.cython
+import mpi4jax.cython  # noqa: F401
```

Comment:
Actually, maybe we can get rid of this line.
Cython shouldn't be a runtime dependency.
My bad I guess

Reply:
WDYM? You need to import the mpi4jax Cython extensions at some point to make sure that the custom calls are registered, but I don't think that makes Cython a runtime dependency.

Reply:
Nevermind.
it's not `import cython`, but `mpi4jax.cython`...